### PR TITLE
test: add version split scenarios

### PR DESCRIPTION
**Pull request by iyerashwin14**

### CODE REVIEW

File: src/test/resources/Features/OCD-2806-SimplifiedEditingOfVersion.feature [hunk 1]
```diff
@@ -45,3 +45,18 @@ Scenario: Verify error message is displayed when the version field is left blank
     And I set the version field to ""
     Then I see error message "Field is required" for version "54"
 
+Scenario Outline: User logs in as ROLE and sees version split link exists on new developer page
+    Given I'm logged in as "<ROLE>"
+    When I navigate to the version edit page for version "49" of product "1482" of developer "934"
```
Comment (andlar) [inline src/test/resources/Features/OCD-2806-SimplifiedEditingOfVersion.feature]:
```suggestion
    When I navigate to the version edit page for version "<version id>" of product "<product id>" of developer "<developer id>"
```

File: src/test/resources/Features/OCD-2806-SimplifiedEditingOfVersion.feature [hunk 2]
```diff
@@ -45,3 +45,18 @@ Scenario: Verify error message is displayed when the version field is left blank
...
+    Then I see split link for version "49"
+    Examples:
+      |ROLE       |
```
Comment (andlar) [inline src/test/resources/Features/OCD-2806-SimplifiedEditingOfVersion.feature]:
```suggestion
      |ROLE       |version id|product id|developer id|
```

File: src/test/resources/Features/OCD-2806-SimplifiedEditingOfVersion.feature [hunk 3]
```diff
@@ -45,3 +45,18 @@ Scenario: Verify error message is displayed when the version field is left blank
...
+      |ROLE_ADMIN |
```
Comment (andlar) [inline src/test/resources/Features/OCD-2806-SimplifiedEditingOfVersion.feature]:
```suggestion
      |ROLE_ADMIN |49 | 1482 | 934 |
```

File: src/test/resources/Features/OCD-2806-SimplifiedEditingOfVersion.feature [hunk 4]
```diff
@@ -45,3 +45,18 @@ Scenario: Verify error message is displayed when the version field is left blank
     And I set the version field to ""
     Then I see error message "Field is required" for version "54"
 
+Scenario Outline: User logs in as ROLE and sees version split link exists on new developer page
+    Given I'm logged in as "<ROLE>"
```
Comment (andlar) [inline src/test/resources/Features/OCD-2806-SimplifiedEditingOfVersion.feature:49]: If you have multiple examples, then all of the variables should be in the table. In the other Scenario in this PR there's only one example, so not having the table is fine

**Review by andlar — CHANGES_REQUESTED** [review]

**Review by andlar — APPROVED** [review]

**Review by ashwinimore — APPROVED** [review]

**Pull request merged**